### PR TITLE
view cart order

### PR DESCRIPTION
**Pull request by ghost**
## Description
View cart in page order
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: public/js/user/viewCartOrder.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  $.each(cartProduct, function(index, value) {
+    total = value.count * value.price;
+    subTotal += total;
+    html += '<tr>\
+      <td class="cart_product"><img src="'+value.img_url+'" alt="Product"></td>\
+      <td class="cart_description">'+value.name+'</td>\
+      <td class="price">'+value.price+'</td>\
+      <td class="qty">'+value.count+'</td>\
+      <td class="price">'+total+'</td>\
+    </tr>';
```
Comment (at-thaitran) [inline public/js/user/viewCartOrder.js]: sửa lại tab

File: public/js/user/viewCartOrder.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  })
+  $('#show-cart').html(html);
+  if(!localStorage.getItem('carts')){
+    var totalSub = '<td colspan="2"><span class="sub-total">0</span> </td>';
+  }
+  else{
+    var totalSub = '<td colspan="2"><span class="sub-total">'+subTotal+'</span> </td>';
+  }
```
Comment (at-thaitran) [inline public/js/user/viewCartOrder.js]: không cần if else chỗ này

File: public/js/user/viewCartOrder.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  htmlSubtotal += '<tr>\
+    <td colspan="2" rowspan="1"></td>\
+    <td colspan="2">'+Lang.get('cart.sub_total')+'</td>\
+    '+totalSub+'\
+  </tr>';
+  $('#sub-total').html(htmlSubtotal);
+  htmlUser += '<div class="col-sm-12">\
+    <p><i class="fa fa-check-circle text-primary"></i>'+Lang.get('order.user.create.your_name')+'<span>'+data_user.name+'</span></p>\
+    <p><i class="fa fa-check-circle text-primary"></i>'+Lang.get('order.user.create.your_email')+'<span>'+data_user.email+'</span></p>\
+    <p><i class="fa fa-check-circle text-primary"></i>'+Lang.get('order.user.create.your_address')+'<span>'+data_user.user_info.address+'</span></p>\
```
Comment (at-thaitran) [inline public/js/user/viewCartOrder.js]: address mình điền vô nên không append trong js

File: public/js/user/viewCartOrder.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  var htmlSubtotal = '';
+  var htmlUser = '';
+  var data_user = JSON.parse(localStorage.getItem('data'));
+  $.each(cartProduct, function(index, value) {
+    total = value.count * value.price;
+    subTotal += total;
+    html += '<tr>\
+      <td class="cart_product"><img src="'+value.img_url+'" alt="Product"></td>\
+      <td class="cart_description">'+value.name+'</td>\
+      <td class="price">'+value.price+'</td>\
```
Comment (at-thaitran) [inline public/js/user/viewCartOrder.js]: thiếu `$`

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**